models/chainlink/avalanche_c/chainlink_avalanche_c_schema.yml

```diff
@@ -1,6 +1,76 @@
 version: 2
 
 models:
+  - name: chainlink_avalanche_c_ccip_onramps
+    meta:
+      blockchain: avalanche_c
+      sector: CCIP
+      project: chainlink
+      contributors: synthquest
+    config:
+      tags: ['avalanche','ccip','chainlink']
+    description: >
+        Chainlink CCIP messages sent from avalanche
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - evt_tx_hash
+            - message
+    columns:
+      - &contract_address
+        name: contract_address
+        description: "avalanche CCIP ramp contract address"
+      - &evt_block_number
+        name: evt_block_number
+        description: "Block number which processed the unique transaction hash"
+      - &evt_block_time
+        name: evt_block_time
+        description: "Timestamp for block event time in UTC"
+      - &evt_index
+        name: evt_index
+        description: "Index value of the transaction"
+      - &evt_tx_hash
+        name: evt_tx_hash
+        description: "Block number which processed the unique transaction hash"
+      - &message
```

Comment:
This column `message` may not be defined in the spell.

Reply:
Responses are json. Probably not appropriate to use for unique column test. I'll remove from test